js/build.js

```diff
@@ -62,36 +65,24 @@ Fliplet.Widget.instance('lock', function(data){
           if (str.length >= 4) {
             if (str === _this.passcode) {
               _this.savePasscodeOnPV(encryptPasscode(_this.passcode));
-              if (Fliplet.Env.get('platform') !== 'web' && _this.touchIdEnabled) {
-                if (window.plugins.touchid) {
-                  window.plugins.touchid.isAvailable(
-                    function(type) {
-                      console.info('Biometric ID available', type);
-                      $lock.find('.bio-id-type').text(type.charAt(0).toUpperCase() + type.slice(1));
-                      $lock.find('.state[data-state=verify]').removeClass('present').addClass('past');
-                      _this.calculateElHeight($lock.find('.state[data-state=touchID]'));
-                      $lock.find('.state[data-state=touchID]').removeClass('future').addClass('present');
-                      $(this).val('');
-                    },
-                    function(message) {
-                      console.info('Biometric ID not available', message);
-                      $lock.find('.state[data-state=verify]').removeClass('present').addClass('past');
-                      _this.calculateElHeight($lock.find('.state[data-state=noTouchID]'));
-                      $lock.find('.state[data-state=noTouchID]').removeClass('future').addClass('present');
-                      $(this).val('');
-                    }
-                  );
-                } else {
+
+              if (_this.touchIdEnabled) {
+                Fliplet.User.Biometrics.isAvailable().then(function(type) {
+                  console.info('Biometric ID available', type);
+
+                  $lock.find('.bio-id-type').text(type.charAt(0).toUpperCase() + type.slice(1));
```

Comment:
@tonytlwu on Android it's up to the actual phone maker to decide the name. It's internally a fingerprint type biometric, but branded differently on each device (e.g. on my Pixel is called "Pixel Imprint". I suggest using the word "Fingerprint" as it seems to be a very common word across vendors (I've done a google search just now).

On web, we don't actually get access to what options the device has (see https://webauthn.guide). The word "biometrics" was added to the code by me as a sample or common value to use since web can use PIN, Face (Windows Hello), USB Keys, Touch ID and more but we don't get to know which one the device is using.

Reply:
See latest changes.